transformer/compose/v3.go

```diff
@@ -80,6 +81,21 @@ func removeNonExistentEnvFilesV3(path string, parsedComposeFile map[string]inter
 							}
 							vals[envFile] = envfiles
 						}
+					} else {
+						checkForDotEnv = true
+					}
+				} else {
+					checkForDotEnv = true
+				}
+				if checkForDotEnv {
```

Comment:
Does Docker compose too use the `.env` file only if there is no environment file specified?